DigitalExcellence/dex-backend
# Filter categories

## Description

Implement the categories in the projectFilter, I also made some changes to the resources and authorization for the category routes so everything is consistent.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Add a project with a category
2. GetAll projects and add the category you added to the project
3. You should only get the project you created

## Link to issue

Closes: #444
